### PR TITLE
Deploy multiple statsd-exporter

### PR DESCRIPTION
**Pull request by artherd42**

### CODE REVIEW

File: prometheus-exporters/statsd-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -1,82 +1,76 @@
...
-      {{- include "statsd-exporter.selectorLabels" . | nindent 6 }}
+      {{- include "statsd-exporter.labels" $exporter.name | nindent 6 }}
   template:
     metadata:
       labels:
-        {{- include "statsd-exporter.selectorLabels" . | nindent 8 }}
+       {{- include "statsd-exporter.labels" $exporter.name | nindent 8 }}
       annotations:
-        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") $ | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/config.yaml") $ | sha256sum }}
```
Comment (thgrs) [inline prometheus-exporters/statsd-exporter/templates/deployment.yaml:24]: shoudn't this remain configmap.yaml?

File: prometheus-exporters/statsd-exporter/values.yaml [hunk 2]
```diff
@@ -1,98 +1,78 @@
...
+      # Prometheus
+      metricsPort: 9102 # The address on which to expose the web interface and generated Prometheus metrics.
+      metricsPath: /metrics # Path under which to expose metrics.
+      prometheus: infra-frontend
+      scrapeInterval: 60s
+      scrapeTimeout: 55s
+
+    - name: 'statsd-exporter-inf'
+      enabled: true
+      udpPort: 9125 # The UDP port on which to receive statsd metric lines.
```
Comment (thgrs) [inline prometheus-exporters/statsd-exporter/values.yaml:49]: just a thought: would it maybe make sense to use a different port here? but maybe it does not make too much sense ...

**Review by thgrs — CHANGES_REQUESTED** [review]

File: prometheus-exporters/statsd-exporter/templates/deployment.yaml (hunk 1 above)
Comment (artherd42) [reply prometheus-exporters/statsd-exporter/templates/deployment.yaml:24]: I have deleted the wrong config file. Just fixed.

File: prometheus-exporters/statsd-exporter/values.yaml (hunk 2 above)
Comment (artherd42) [reply prometheus-exporters/statsd-exporter/values.yaml:49]: Doesn't matter since each has it's own ingress,

**Pull request merged**